optics-core/src/Optics/Indexed.hs

```diff
@@ -1,13 +1,13 @@
 module Optics.Indexed
   (
   -- * Functors with index
-    FunctorWithIndex (..)
+    IxFunctpr (..)
```

Comment:
Functpr

Is renaming to avoid clashes with `lens`?

Reply:
Not really, I just figured that since we have IxProfunctor and indexed optics start with Ix, classes can be the same for consistency.